packages/datagateway-download/public/datagateway-download-settings.example.json

```diff
@@ -62,5 +62,6 @@
       "order": 1
     }
   ],
-  "pluginHost": "http://localhost:3000"
+  "pluginHost": "http://localhost:3000",
+  "uiFeatures": ["DOWNLOAD_PROGRESS"]
```

Comment:
A very minor point, but since this is JSON & Javascript normally the convention is `camelCase` and so `downloadProgress` may make more sense. Also, I dislike that this is just an array which needs a specific string - would an object which has boolean properties be clearer and less likely for someone to mess up by mispelling something?
i.e.

```
"uiFeatures": {
  "downloadProgress": true
}
```

If we did do things like that - I would then suggest that `downloadProgress` be disabled by default as this is a DLS only feature

Reply:
Since it is a JSON file, I don't think the IDE would provide suggestion for the key anyways (unless we create a JSON schema). In TypeScript, I typed the array as an array of a union string, so in terms of autocompletion I think both are the same.

Reply:
Yes, but from the perspective from a sysadmin who won't necessarily understand the source code - if they had a settings file with `"uiFeatures: []"` - how would they know what strings to put in? If this was `"uiFeatures": {"downloadProgress":false}` - they'd know to merely flip the boolean

Reply:
Ahh, I see what you mean now. I agree with you, I'll change it to an object type.